docs/introduction/Car-registry-tutorial.rst

```diff
@@ -202,9 +207,9 @@ Implementation of CarBoxDataSerializer:
 Implementation of CarBox:
 *************************
 
- ``CarBox`` is implemented according to description from ``Custom Box Class creation`` section as ``public class CarBox extends AbstractNoncedBox<PublicKey25519Proposition, CarBoxData, CarBox>``
+ ``CarBox`` is implemented according to the description from ``Custom Box Class creation`` section as ``public class CarBox extends AbstractNoncedBox<PublicKey25519Proposition, CarBoxData, CarBox>``
```

Comment:
@zennermac 

A ``CarBox`` is implemented according to the description from the ``Custom Box Class Creation`` section as a ``public class CarBox extends AbstractNoncedBox<PublicKey25519Proposition, CarBoxData, CarBox>``